src/tests/fixtures/env/storage.json

```diff
@@ -135,7 +135,8 @@
 					"body": "",
 					"action": "view",
 					"meta": {
-						"txId": "ea63bf9a4b3eaf75a1dfff721967c45dce64eb7facf1aef29461868681b5c79b"
+						"txId": "ea63bf9a4b3eaf75a1dfff721967c45dce64eb7facf1aef29461868681b5c79b",
```

Comment:
Avoid abbreviations like `txId`. Always use written out versions like `transactionId` that don't need further explanation.